beetmoverscript/docker.d/init_worker.sh

```diff
@@ -21,6 +21,7 @@ case $COT_PRODUCT in
         test_var_set 'DEP_PARTNER_KEY'
         test_var_set 'MAVEN_ID'
         test_var_set 'MAVEN_KEY'
+        test_var_set 'PROJECT_NAME'
```

Comment:
more details about this env var [here](https://mozilla-hub.atlassian.net/browse/SVCSE-1067)

Reply:
This is not what I thought... looked in the `configmap` and the value is `beetmover`